# action_type = (0 : Request Only Target, 1 : Request Target and Data, 2 : Request Only Data)
int32 action_type
float32 control_signal_ankle
float32 control_signal_knee
---
float32 position_feedback_ankle
float32 position_feedback_knee
float32 velocity_feedback_ankle
float32 velocity_feedback_knee
float32 torque_feedback_ankle
float32 torque_feedback_knee
